### PR TITLE
Taskreview

### PR DESCRIPTION
**Pull request by Bart-Ilg**
merging in task review changes

### CODE REVIEW

File: src/Components/TaskReview/PercentageBar.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+   function percentageBarRightStyle() {
+      return {
+          'width' : (((100 - perc)/100) * percentageBarWidth).toString() + 'px',
+         'backgroundColor' : rightColor,
+         'borderBottomRightRadius' : '12px',
+         'borderTopRightRadius' : '12px'
+      };
+   }
+
```
Comment (ryanmadamba) [inline src/Components/TaskReview/PercentageBar.js:32]: Good using functions for the styling here

**Review by ryanmadamba — APPROVED** [review]
Looks great, Bart!

File: src/Components/TaskReview/TaskReview.js [hunk 2]
```diff
@@ -67,13 +55,26 @@ export default function TaskReview(props) {
...
       />);
     }
+    else if(compDisplayed === 'get-help') {
+      return(<TaskReviewHelp/>);
+    }
+    else if(compDisplayed === 'task-results') {
+      return(<TaskReviewResults 
+        submission = {submission}
+      />);
+    }
```
Comment (ConnorBoulais) [inline src/Components/TaskReview/TaskReview.js:65]: clean, organized logic here

File: src/Components/TaskReview/TaskReviewResults.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   return (
+        <div className="row">
+          <div className="column"></div>
+          <div className="column">
+            <div className="stat-box">
+              <TaskReviewStats submission={submission} />
+            </div>
+          </div>
+          <div className="column">
+            <div className="objective-column">
```
Comment (ConnorBoulais) [inline src/Components/TaskReview/TaskReviewResults.js:33]: Stats look really good, nice job

**Review by ConnorBoulais — APPROVED** [review]
Great job on task review here, it looks awesome!

**Pull request merged**